packages/extensions/openapi-to-typespec/src/transforms/transform-arm-resources.ts

```diff
@@ -515,7 +489,11 @@ function convertResourceOtherGetOperations(
         const swaggerOperation = operations[operation.OperationID];
         if (swaggerOperation.requests && swaggerOperation.requests[0]) {
           const op = transformRequest(swaggerOperation.requests[0], swaggerOperation, getSession().model);
-          op.operationGroupName = getOperationGroupName(operation.OperationID, resourceMetadata.Name);
+          op.operationGroupName = getOperationGroupName(operation.OperationID);
+          if (!op.fixMe) {
+            op.fixMe = [];
+          }
+          op.fixMe.push(`// FIXME: ${operation.OperationID} could not be converted to a resource operation`);
```

Comment:
For the operations that could not be converted, could you also output them into console log?

Reply:
though it will not be converted to a resource operation, it will be converted to a normal operation, this comment aims to help user to double check the result.